### DISCUSSION TITLE
Customized margin & padding size not possible.

### DISCUSSION THREAD

**Discussion by noisrucer** [Textualize/textual#2066] [Q&A] [answered]
Is there any way that I can customize my padding and margin? Also, I want to get rid of the default margin between cells in a grid layout. I've tried all possible ways but I didn't come to a solution.

**Comment by davep** [comment, marked as answer]
> Is there any way that I can customize my padding and margin?

Textual's CSS has both [padding](https://textual.textualize.io/styles/padding/) and [margin](https://textual.textualize.io/styles/margin/) properties.

> Also, I want to get rid of the default margin between cells in a grid layout.

The grid [has a number of properties for changing its appearance](https://textual.textualize.io/styles/grid/), amongst them is [`grid-gutter`](https://textual.textualize.io/styles/grid/grid_gutter/) which sounds to be what you're looking for.

> I've tried all possible ways but I didn't come to a solution.

It would be helpful if you gave some indication as to what ways you did try and how they didn't work for you. The above would seem to be what you're looking for, so if they didn't work for you, a small stand-alone example of what you're trying with an explanation of what you're wanting to do and how it's failing will make it easy to help.
